test/generators/shop_model_generator_test.rb

```diff
@@ -29,6 +33,13 @@ class ShopModelGeneratorTest < Rails::Generators::TestCase
     end
   end
 
+  test "injects into application controller" do
+    run_generator
+    assert_file "app/controllers/application_controller.rb" do |controller|
+      assert_match "def shop", controller
+    end
```

Comment:
Ok. I have no idea what that means, but I will persist in poking around with this neat-o app. I just want to know the shop logged in when creating an App.. seems I have to dig into this concept of the engine...